src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -30,7 +30,7 @@
                                AllowFallbackOnTargetSelection="true"
                                IncludeFrameworkReferences="false"
                                NuGetPackagesDirectory="$(PackagesDir)"
-                               RuntimeIdentifier="win7-x86"
```

Comment:
Can you make this an overrideable property? I'm fine setting it to win7-x64 but it would allow people to also set it to win7-x86.